kernel/kernel-jammy-src/6.0/0001-kernel-enable-HID_SENSOR.patch

```diff
@@ -0,0 +1,48 @@
+From 8abf5719599328104b1eb0c6cbc5f9526f106adc Mon Sep 17 00:00:00 2001
```

Comment:
How is this integrated into our build script?
Currently as I understand it's not?

Reply:
https://github.com/IntelRealSense/realsense_mipi_platform_driver/blob/2f918beb8d806b0d8be3a3d468a235a0dd1d7190/apply_patches.sh#L55